The site fails with error 500

https://anatoly-scherbakov.github.io/geopatterns-demo/site/ shows an error 500 for me.

```
[ERROR] Runtime.ImportModuleError: Unable to import module 'geopatterns_demo.api': No module named 'loremipsum'Traceback (most recent call last): | [ERROR] Runtime.ImportModuleError: Unable to import module 'geopatterns_demo.api': No module named 'loremipsum' Traceback (most recent call last):
```

The `loremipsum` module is not included into the installation package. Please include it into `pyproject.toml` by issuing:

```shell
poetry add loremipsum
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -12,6 +12,7 @@ typer = "^0.3.2"
 fastapi = "^0.65.2"
 mangum = "^0.11.0"
 sentry-sdk = "^1.0.0"
+py-loremipsum = "^1.1.0"
 
 
 [tool.poetry.dev-dependencies]

